Runtimes/Core/runtime/CMakeLists.txt

```diff
@@ -149,13 +149,17 @@ if("${SwiftCore_OBJECT_FORMAT}" STREQUAL "elfx")
     "${SwiftCore_SWIFTC_SOURCE_DIR}/include"
     "${PROJECT_BINARY_DIR}/include")
   target_link_libraries(swiftrt PRIVATE swiftShims)
-  install(TARGETS swiftrt DESTINATION "${CMAKE_INSTALL_LIBDIR}/swift")
+  install(FILES $<TARGET_OBJECTS:swiftrt>
+    DESTINATION "${CMAKE_INSTALL_LIBDIR}/swift/${SwiftCore_PLATFORM_SUBDIR}/${SwiftCore_ARCH_SUBDIR}"
```

Comment:
`SwiftCore_PLATFORM_SUBDIR` and `SwiftCore_ARCH_SUBDIR` are currently only required if your platform has a filesystem. If we're going to start requiring it in more places, is there something we can do to at least derive a reasonable default from what the compiler is expecting?

Reply:
These values are what the compiler is expecting. We can try to replicate the current switch from the SwiftSupport if you like.